activesupport/lib/active_support/event_reporter/test_helper.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module ActiveSupport::EventReporter::TestHelper # :nodoc:
+  class EventSubscriber
```

Comment:
```suggestion
  class EventSubscriber # :nodoc:
```

https://shopify-ac-structured-events.rails-docs-preview.pages.dev/api/classes/ActiveSupport/EventReporter/TestHelper/EventSubscriber